datalad/utils.py

```diff
@@ -2279,5 +2280,13 @@ def get_suggestions_msg(values, known, sep="\n        "):
     return ''
 
 
+def maybe_shlex_quote(val):
+    """
+    shlex_quote() a value if the command is not run on a Windows
+    machine.
+    """
+
+    return val if on_windows else shlex_quote(val)
+
 
-lgr.log(5, "Done importing datalad.utils")
+lgr.log(5, "Done importing datalad.utils")
```

Comment:
I personally prefer line endings all the way to the last line, but we are not agreement within the project already.

Reply:
done.